src/main/java/org/sagebionetworks/web/client/SynapseClient.java

```diff
@@ -225,8 +225,6 @@ Boolean isAliasAvailable(String alias, String aliasType)
   HashMap<String, WikiPageKey> getPageNameToWikiKeyMap()
     throws RestServiceException;
 
-  String deleteApiKey() throws RestServiceException;
```

Comment:
This called `SynapseJavaClient.deleteApiKey`, which was removed since the last time we updated `SynapseJavaClient`.